resources/requirements_mypy.txt

```
@@ -1,3 +1,4 @@
 mypy==0.982
 types-PyYAML==6.0.12.1
 types-setuptools==65.5.0.2
+types-requests
```

We require here pinning requirements 
```suggestion
types-requests==2.28.11.15 
```